plasmapy/utils/tests/test_units_helpers.py

```diff
@@ -6,7 +6,7 @@
 from astropy.constants import c, m_e
```

Comment:
The module docstring still uses `plasmapy.utils.units_helpers`, instead of `plasmapy.utils._units_helpers`.

Reply:
Done.